docs/userGuide/components/dropdown.md

```diff
@@ -59,7 +59,7 @@
           <li><a href="#dropdown" class="dropdown-item">Separated link</a></li>
         </ul>
       </dropdown>
-      <a href="#dropdown" class="btn btn-default" role="button">Right</a>
+      <a href="#dropdown" class="btn btn-light w-100" role="button">Right</a>

```

Comment:
The buttons in the button group blends in with the background colour of our `tip-box` component. Let's change it to `btn-secondary` or have a "traffic light" theme `danger`, `warning`, `success`.